### ISSUE TITLE
Is this plugin compatible with wordpress multisite installation?

### ISSUE THREAD

**Issue by cyrusd87** [benhuson/Taxonomy-Images#31]
Hi all,

I have a question about this plugin.. Is this plugin compatible with wordpress in multisite configuration? Are there performance issues? 

Thanks

**Comment by benhuson** [comment]
Yes it is compatible. I've used in several multisite installs without performance issues.
